src/gui/mainwindow.cpp

```diff
@@ -143,6 +152,13 @@ MainWindow::MainWindow(QWidget *parent)
 {
     m_ui->setupUi(this);
 
+    Theme::Serialization::registerGuiColorProviders();
+    Theme::Serialization::registerGuiFontProviders();
```

Comment:
Aren't `registerFontProviders` and `registerColorProviders` enough? (without `Gui` part) Or there could be non-GUI providers?